ts/features/itwallet/presentation/remote/screens/ItwRemoteFailureScreen.tsx

```diff
@@ -103,6 +103,30 @@ const ContentView = ({ failure }: ContentViewProps) => {
             }
           };
         }
+        case RemoteFailureType.EID_EXPIRED: {
+          return {
+            title: I18n.t(
+              "features.itWallet.presentation.remote.eidExpiredScreen.title"
+            ),
+            subtitle: I18n.t(
+              "features.itWallet.presentation.remote.eidExpiredScreen.subtitle"
+            ),
+            pictogram: "identityRefresh",
+            action: {
+              label: I18n.t(
+                "features.itWallet.presentation.remote.eidExpiredScreen.primaryAction"
+              ),
+              onPress: () =>
+                machineRef.send({ type: "go-to-identification-mode" })
+            },
+            secondaryAction: {
+              label: I18n.t(
+                "features.itWallet.presentation.remote.eidExpiredScreen.secondaryAction"
+              ),
+              onPress: () => machineRef.send({ type: "go-to-wallet" })
```

Comment:
In this case we must close the flow, this will make the navigation to use the correct animation.
```suggestion
              onPress: () => machineRef.send({ type: "close" })
```

Reply:
Hi @mastro993 , just to understand better, could you explain why it is not okay to go to the wallet? As it was done in the other cases ( missing credentials) and required by design